foundation/foundation-upgrade/src/test/java/com/boozallen/aissemble/upgrade/migration/AbstractMigrationTest.java

```diff
@@ -22,6 +25,14 @@ public abstract class AbstractMigrationTest {
 
     private static final String TEST_FILES_FOLDER = Paths.get("target", "test-classes", "test-files").toString();
 
+    protected static Boolean validateMigration(File original, File migrated) {
```

Comment:
A: Use primitive `boolean` here

Reply:
Updated to use primitive `boolean` type.